SolWatch/Game1.cs

```diff
@@ -81,6 +80,7 @@ protected override void Update(GameTime gameTime)
         protected override void Draw(GameTime gameTime)
         {
             GraphicsDevice.Clear(Color.Black);
+            var celestialBodySymbolSize = new Point(celestialBodySymbolLength, celestialBodySymbolLength);
```

Comment:
What's the idea behind removing this at the top, then redefining it down here? Doesn't it just make 2 lines of code where 1 line we started with would have worked equally well?

Reply:
Agreement: use celestialBodySymbolSize as a read-only point at the top.